singleheader/CMakeLists.txt

```diff
@@ -17,6 +17,36 @@ set(SINGLEHEADER_REPO_FILES
 )
 set_source_files_properties(${SINGLEHEADER_FILES} PROPERTIES GENERATED TRUE)
 
+#
+# Build the library
+#
+if(SIMDJSON_BUILD_STATIC)
+  MESSAGE( STATUS "Building a static library." )
+  add_library(simdjson STATIC "")
```

Comment:
Hmm. I actually think we'd want simdjson to be built from src/ as well, rather than singleheader, so that any debugging stuff relates to the actual files/lines. simdjson.cpp is "just an export," and we already have a test that it works in singleheader (aggregate_demo).

Also, this means we're not testing or benchmarking the actual library we're building, which is problematic.